README.md

```diff
@@ -1,7 +1,8 @@
 # Parasoft Findings for Azure DevOps
 
-The Parasoft Findings extension is designed to be used in a post test-execution build step that integrates static analysis, functional and unit test reports generated by Parasoft tools into your Azure DevOps project. The task will collect static analysis and test results from the Parasoft report XML files.
+The Parasoft Findings extension is designed to be used in a post test-execution build step that integrates static analysis, functional and unit test reports generated by Parasoft tools into your Azure DevOps project. The task will collect static analysis, test results and code coverage from the Parasoft report XML files.
```

Comment:
Should we make the orders for static analysis, test results and code coverage here consistent with line 4-6?